src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,8 +23,10 @@ class Lambda {
       (event.itemType, event.eventType) match {
         case (ItemType.Content, EventType.Delete) =>
           sendPurgeRequest(event.payloadId)
+        case (ItemType.Content, EventType.Update) =>
+          sendSoftPurgeRequest(event.payloadId)
         case other =>
-          // for now we only send purges for content takedowns, so ignore any other events
+          // for now we only send purges for content, so ignore any other events
```

Comment:
@johnduffell At the moment we support event types of `Update`, `Delete` and `RetrievableUpdate`. You can see them here: https://github.com/guardian/content-api-models/blob/master/models/src/main/thrift/events/crier/event.thrift

@gklopper I think you will also want to include `RetrievableUpdate`. This is used when the content is too large and exceeds the kinesis limits. In the event of this, we change send this new event type with a payload containing the following: https://github.com/guardian/content-api-models/blob/master/models/src/main/thrift/events/crier/event.thrift#L24 (allowing the client to retrieve the item themselves). This typically occurs for live blogs, but is not exclusive of.